akka-http-testkit/src/main/scala/akka/http/scaladsl/testkit/RouteTest.scala

```diff
@@ -54,6 +53,9 @@ trait RouteTest extends RequestBuilding with WSTestRequestBuilding with RouteTes
     if (dynRR.value ne null) dynRR.value
     else sys.error("This value is only available inside of a `check` construct!")
 
+  implicit def routeTestTimeout2Timeout(implicit timeout: RouteTestTimeout): Timeout =
```

Comment:
Would an opposite conversion perhaps be simpler?
If I provided a timeout, and no route result timeout, the route result timeout could be inferred from the timeout, and if I want the route result to be more specific, I would provide it explicitly - thus it has precedence over the normal timeout.

WDYT?